### PR TITLE
MySQL/HowTo: Fix small formatting

### PR DESCRIPTION
**Pull request by laysauchoa**
So it can be easily maintained in the future

### CODE REVIEW

File: docs/products/mysql/howto/migrate-from-external-mysql.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 Requirements
-----------------
+------------
 
 To perform a migration from an external MySQL to Aiven for MySQL the following requirements need to be satisfied:
 
 * The source server needs to be publicly available or accessible via a virtual private cloud (VPC) peering connection between the private networks, and any firewalls need to be open to allow traffic between the source and target servers.
-* You have user account on the source server with sufficient privileges to create a user for the replication process.
+* You have an user account on the source server with sufficient privileges to create a user for the replication process.
```
Comment (kevinbowman-aiven) [inline docs/products/mysql/howto/migrate-from-external-mysql.rst:12]: Should be "a", as in "You have a user account"

**Review by kevinbowman-aiven — CHANGES_REQUESTED** [review]
Just a tiny typo fix, everything else looks good to me.

**Comment by laysauchoa** [comment]
thanks @kevinbowman-aiven I've fixed now!

**Review by kevinbowman-aiven — APPROVED** [review]
Looks good to me!

**Pull request merged**